README.md

```diff
@@ -20,7 +20,7 @@ To get started, sign up [here](https://clevertap.com/live-product-demo/).
 ## ✅ Supported Versions
 
 - [CleverTap Android SDK version 4.4.0](https://github.com/CleverTap/clevertap-android-sdk/releases/tag/core-v4.4.0)
-- [CleverTap iOS SDK version 3.10.0](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/3.10.0)
+- [CleverTap iOS SDK version 4.1.0](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/4.1.0)
```

Comment:
@akashvercetti CHANGELOG says the supported iOS version is 4.1.1
README says the supported iOS version is 4.1.0.
Fix the one that's necessary.

Reply:
done